lib/DataStore.js

```diff
@@ -550,6 +550,25 @@ DataStore.prototype.storeIrcClientConfig = function(config) {
     });
 };
 
+DataStore.prototype.getUserFeatures = function(userId) {
+    return this._userStore.getMatrixUser(userId).then((matrixUser) => {
+        if (!matrixUser) {
+            return {};
+        }
+        return matrixUser.get("features") || {};
+    });
+};
+
+DataStore.prototype.storeUserFeatures = function(userId, features) {
```

Comment:
I'm a bit conflicted on the name here (store vs set), but that's an argument for another day